Resources/Prototypes/Roles/Jobs/Fun/visitors_startinggear.yml

```
@@ -299,7 +299,7 @@
     eyes: ClothingEyesGlassesSecurity
     outerClothing: ClothingOuterCoatDetectiveLoadout
     pocket1: ForensicScanner
-    pocket2: ForensicPad
+    pocket2: WeaponRevolverSpearheadBlack # Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)